src/PrestaShopBundle/Resources/views/Admin/Configure/AdvancedParameters/Blocks/import_panel.html.twig

```diff
@@ -0,0 +1,178 @@
+                <p class="alert-text js-import-file"></p>
+            </div>
+
+            {% block import_file_history_block %}
+                {% include '@PrestaShop/Admin/Configure/AdvancedParameters/Blocks/import_file_history.html.twig' %}
+            {% endblock %}
+
+            <hr>
+
+            <div class="form-group">
+                {{ ps.label_with_help(('Language of the file'|trans), ('The locale must be installed'|trans({}, 'messages'))) }}
+                {{ form_errors(importForm.iso_lang) }}
+                {{ form_widget(importForm.iso_lang) }}
+            </div>
+
+            <div class="form-group">
+                <label class="form-control-label">{{ 'Field separator'|trans }}</label>
+                {{ form_errors(importForm.separator) }}
+                {{ form_widget(importForm.separator) }}
+                <small class="form-text">{{ 'e.g. '|trans({}, 'Admin.Advparameters.Help') }} 1; Blouse; 129.90; 5</small>
+            </div>
+
+            <div class="form-group">
+                <label class="form-control-label">{{ 'Multiple value separator'|trans({}, 'messages') }}</label>
+                {{ form_errors(importForm.multiple_value_separator) }}
+                {{ form_widget(importForm.multiple_value_separator) }}
+                <small class="form-text">{{ 'e.g. '|trans({}, 'Admin.Advparameters.Help') }} Blouse; red.jpg, blue.jpg, green.jpg; 129.90</small>
+            </div>
+
+            <hr>
+
+            <div class="form-group js-truncate-form-group">
+                <label class="form-control-label">
+                    {{ 'Delete all [1]categories[/1] before import'|trans({'[1]': '<span class="js-entity-name">', '[/1]': '</span>'})|raw }}
+                </label>
+                {{ form_errors(importForm.truncate) }}
+                {{ form_widget(importForm.truncate) }}
+            </div>
+
+            <div class="form-group js-match-ref-form-group">
+                {{ ps.label_with_help(('Use product reference as key'|trans), ('If enabled, the product\'s reference number MUST be unique!'|trans({}, 'Admin.Advparameters.Help'))) }}
+                {{ form_errors(importForm.match_ref) }}
+                {{ form_widget(importForm.match_ref) }}
+            </div>
+
+            <div class="form-group js-regenerate-form-group">
+                <label class="form-control-label">{{ 'Skip thumbnails regeneration'|trans }}</label>
+                {{ form_errors(importForm.regenerate) }}
+                {{ form_widget(importForm.regenerate) }}
+            </div>
+
+            <div class="form-group js-force-ids-form-group">
+                {{ ps.label_with_help(('Force all ID numbers'|trans), ('If you enable this option, your imported items\' ID number will be used as-is. If you do not enable this option, the imported ID numbers will be ignored, and PrestaShop will instead create auto-incremented ID numbers for all the imported items.'|trans({}, 'Admin.Advparameters.Help'))) }}
```

Comment:
Delete the '\' sign, I think. And remove the '-' in 'as-is', it should be 'as is'.

Reply:
@sarjon don't forget to update this one too :)